DQMOffline/Configuration/python/autoDQM.py

```diff
@@ -36,11 +40,11 @@
 
 	    'stripZeroBias': ['DQMOfflineTrackerStripMinBias',
 			      'PostDQMOffline',
-                              'DQMHarvestTrackerStrip+DQMCertTrackerStrip'],
```

Comment:
why is `DQMCertTrackerStrip` removed?

Reply:
This module was giving an error without crash, so I originally removed it until I found another solution. I am very sorry for the inconvenience. Thanks for the check!